.travis.yml

```diff
@@ -14,3 +14,25 @@ script:
 - "./debuild.sh |& tee log.txt"
 notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+env:
+  global:
+  - PACKAGE_NAME=meilix-systemlock
+
+before_install:
+- export BUILD_VERSION=$(date "+%Y%m%d").$TRAVIS_BUILD_NUMBER
+- export PACKAGE_NAME_VERSION=$PACKAGE_NAME.$BUILD_VERSION.deb
+
+script:
+- dpkg-deb --build $PACKAGE_NAME
+- mv $PACKAGE_NAME.deb $PACKAGE_NAME_VERSION
```

Comment:
The package name is controlled by the debian files and not required to be changed here for now
I think you should open a clear issue of what you want to do 
Because it is really difficult if the idea of what you want to do is not clear?